lib/assert.js

```diff
@@ -306,10 +301,38 @@ assert.doesNotThrow = function doesNotThrow(block, error, message) {
       expected: error,
       operator: 'doesNotThrow',
       message: `Got unwanted exception${details}\n${actual.message}`,
-      stackStartFn: doesNotThrow
+      stackStartFn: assert.doesNotThrow
```

Comment:
Maybe name `assert.doesNotThrow` as a top level function and reference it directly here to avoid being messed up by monkey-patched `assert.doesNotThrow`